ui/widgets/nuxeo-selectivity.js

```diff
@@ -7137,25 +7137,18 @@ typedArrayTags[weakMapTag] = false;
           ),
 
           singleSelectedItem: (opts) => (
-            `${'<span class="selectivity-single-selected-item" ' +
-            'data-item-id="'}${escapeHTML(opts.id)}">${
-              opts.removable ? '<a class="preserve-white-space selectivity-single-selected-item-remove">' +
-                  '<span class="selectivity-remove"></span>' +
-                  '</a>'
-                : ''
-            }${this.selectionFormatter(opts.item || opts)}</span>`
+            `<span class="selectivity-single-selected-item"
+            data-item-id="${escapeHTML(opts.id)}">${opts.removable ? '<a class="preserve-white-space selectivity-single-selected-item-remove">'+
+            '<span class="selectivity-remove" role="button" aria-label="'+ this._computeAriaLabel() + '"></span></a>' : ''}${this.selectionFormatter(opts.item || opts)}</span>`
```

Comment:
Since you are already in a string literal, why not take advantage of it:

```js
<span class="selectivity-remove" role="button" aria-label="${this._computeAriaLabel()}"></span>
```

Reply:
(see comment on the method `_compute ArialLabel()`)

Reply:
Done